app/models/candidate_interface/satisfaction_survey_form.rb

```diff
@@ -4,12 +4,35 @@ class SatisfactionSurveyForm
 
     attr_accessor :question, :answer
 
+    QUESTIONS_WE_ASK = [
+      'I would recommend this service to a friend or colleague',
+      'I found this service unnecessarily complex',
+    ].freeze
+
     validates :question, presence: true
+    validates :question, inclusion: { in: QUESTIONS_WE_ASK, allow_blank: false }
```

Comment:
I think we need to validate that the `answer` is included in what we expect as well.

Reply:
Also, what do these errors show up as to the user? Do we need some localisation strings?

Reply:
There are free text answers too, but we could definitely validate certain questions have certain answers @tvararu

Reply:
Yea good point regarding the errors. They currently don't show up and it just renders the view. The only way that page can be triggered is if they manually change the hidden field for the question. I'm not sure what the error message should be? 

Perhaps 'Stop it please' haha. But seriously, maybe 'Question was not recognised. Please input your answer again.'?

Reply:
This is longer necessary as we don;t allow users to mess with the question anymore since we use the params to get the question rather than the hidden field.